metrics/measured_http/http.go

```diff
@@ -55,17 +58,34 @@ func New(m serveMux, clk clock.Clock, stats prometheus.Registerer, opts ...otelh
 		},
 		[]string{"endpoint", "method", "code"})
 	stats.MustRegister(responseTime)
+
+	inFlightRequestsGauge := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "in_flight_requests",
+			Help: "Tracks the number of WFE requests currently in flight, labeled by endpoint.",
+		},
+		[]string{"method"},
+	)
+	stats.MustRegister(inFlightRequestsGauge)
+
 	return otelhttp.NewHandler(&MeasuredHandler{
-		serveMux: m,
-		clk:      clk,
-		stat:     responseTime,
+		serveMux:              m,
+		clk:                   clk,
+		stat:                  responseTime,
+		inFlightRequestsGauge: inFlightRequestsGauge,
 	}, "server", opts...)
 }
 
 func (h *MeasuredHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	begin := h.clk.Now()
 	rwws := &responseWriterWithStatus{w, 0}
 
+	subHandler, pattern := h.Handler(r)
+	if pattern != "/" {
```

Comment:
Why the exclusion for `"/"`?

Reply:
We're mostly concerned about endpoints that dispatch RPCs, I knew we could safely discard "/". We could probably discard others as well.

Reply:
Fair enough. I think on balance, having fewer lines lines of code is slightly more appealing than economizing on the one timeseries, but I can really go either way on this one.

Reply:
Agreed. I think that excluding "/" makes sense if this were implemented in the WFE, where we could also exclude other endpoints we don't care about, like "/build". But here in this helper library that may be used by other webservers that _do_ care about the root path, excluding it is likely to lead to confusion in the future.